CMakeLists.txt

```diff
@@ -248,6 +252,32 @@ IF (ASPECT_USE_FP_EXCEPTIONS)
   ENDIF()
 ENDIF()
 
+# Check if we want to use cotire to precompile header files and prepare a
+# unity build target. This speeds up compile time, but can fail on some
+# machines, therefore it is disabled by default.
+SET(ASPECT_PRECOMPILE_HEADERS OFF CACHE BOOL "Precompile external header files using cotire to speedup compile time.")
+
+IF (ASPECT_PRECOMPILE_HEADERS)
+  MESSAGE(STATUS "Enabling cotire to precompile external header files.")
+  INCLUDE(cmake/cotire)
+
+# Set some properties that are necessary for the header precompilations 
+# and unity build to pass.
+# E.g. we need to exclude some headers that frequently cause problems, 
+# and undefine some macros between source files that would otherwise 
+# interfere with each other.
+  set_target_properties (aspect PROPERTIES
+      COTIRE_PREFIX_HEADER_IGNORE_PATH
+      "/usr/include/;${CMAKE_SOURCE_DIR};${CMAKE_BINARY_DIR}")
+  set_source_files_properties (${TARGET_SRC} PROPERTIES
+      COTIRE_UNITY_SOURCE_POST_UNDEFS "INSTANTIATE")
+# This line activates the cotire module for the aspect target, 
+# and therefore the whole precompilation
+  cotire(aspect)
+ELSE()
+  MESSAGE(STATUS "Disabling cotire. Do not precompile external header files.")
```

Comment:
Just omit the second part of the sentence. It reads like a command to the reader ("do not do this" -- to which I would reply "do what?")